src/applications/ivc-champva/10-7959C/config/submitTransformer.js

```diff
@@ -22,8 +21,10 @@ export default function transformForSubmit(formConfig, form) {
   copyOfData.hasOtherHealthInsurance =
     copyOfData.applicantHasPrimary || copyOfData.applicantHasSecondary;
 
-  copyOfData.applicantName.middle =
-    copyOfData.applicantName?.middle?.charAt(0) ?? '';
+  if (copyOfData.applicantName?.middle) {
+    copyOfData.applicantName.middle =
+      copyOfData.applicantName?.middle?.charAt(0) ?? '';
+  }
```

Comment:
Fixes issue where we would error out if no middle name was provided upon submission (only impacts E2E tests)